pkg/supply/env/config.go

```diff
@@ -20,6 +20,35 @@ type amsDataDeprecated struct {
 	Root string `json:"root"`
 }
 
+type VcapApplication struct {
+	ApplicationID   string   `json:"application_id"`
+	ApplicationName string   `json:"application_name"`
+	ApplicationUris []string `json:"application_uris"`
+	CfAPI           string   `json:"cf_api"`
+	Limits          struct {
+		Fds int `json:"fds"`
+	} `json:"limits"`
+	Name             string   `json:"name"`
+	OrganizationID   string   `json:"organization_id"`
+	OrganizationName string   `json:"organization_name"`
+	SpaceID          string   `json:"space_id"`
+	SpaceName        string   `json:"space_name"`
+	Uris             []string `json:"uris"`
+	Users            any      `json:"users"`
```

Comment:
From what I can see, only the `ApplicationName` is used in the coding. Let's strip down the struct to what we need exactly, no need to parse the other values.